package.json

```diff
@@ -66,7 +67,10 @@
     "@react-native-firebase/crashlytics": "^12.3.0",
     "@react-native-firebase/perf": "^12.3.0",
     "@react-native-picker/picker": "^2.4.3",
+    "@react-navigation/core": "^6.4.8",
```

Comment:
@blazejkustra Why are we adding this dependencies?

- @react-navigation/core
- @react-navigation/drawer
- @react-navigation/routers
- react-freeze
- react-native-animatable
- js-yaml

Reply:
These libraries are imported in the project but not directly installed. After navigation update I think `@react-navigation/drawer` and `@react-navigation/routers` are no longer required. I'll uninstall them.